### PR TITLE
Put info with migration help

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

**Comment by ryestew** [comment]
Here is a version with some updated formatting and text:
image

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js [hunk 1]
```diff
@@ -466,9 +472,38 @@ export class LandingPage extends ViewPlugin {
           <div class="d-flex justify-content-between">
             <div class="d-flex flex-column">
               <div class="border-bottom d-flex justify-content-between clearfix py-3 mb-4">
-                <div class="mx-4 w-100">
+                <div class="mx-4 w-100 d-flex">
                   ${img}
                   <audio id="remiAudio" muted=false src="assets/audio/remiGuitar-single-power-chord-A-minor.wav"></audio>
+                  <div class="w-80 pl-5 ml-5">
```
Comment (ryestew) [inline apps/remix-ide/src/app/ui/landing-page/landing-page.js:478]: Here are 2 updates:
```
<div class="w-80 pl-5 ml-5"><h4 class="mb-1">Quicklinks</h4><a href="https://medium.com/remix-ide/migrating-files-to-workspaces-8e34737c751c?source=friends_link&amp;sk=b75cfd9093aa23c78be13cce49e4a5e8">Guide for migrating</a> your files to a File Explorer workspace
<h6 class="mt-1 mb-1">Migration tools:</h6><li><u class="text_2A2Ew px-1">Download all Files</u></li><li><u class="text_2A2Ew px-1">Restore Backup Zip</u>- a plugin to current workspace</li> <h6 class="mt-1 mb-0">Help</h6><li><a target="__blank" href="https://gitter.im/ethereum/remix" class="text_2A2Ew mx-1">Gitter channel</a></li><li><a target="__blank" href="https://github.com/ethereum/remix-project/issues" class="text_2A2Ew mx-1">Report on Github</a></li></div>
```
or without bullet points
```
<div class="w-80 pl-5 ml-5"><h4 class="mb-1">Quicklinks</h4><a href="https://medium.com/remix-ide/migrating-files-to-workspaces-8e34737c751c?source=friends_link&amp;sk=b75cfd9093aa23c78be13cce49e4a5e8">Guide for migrating</a> your files to a File Explorer workspace
<h6 class="mt-1 mb-1">Migration tools:</h6><p class="mb-0"><u class="text_2A2Ew px-1">Download all Files</u></p><p class="mb-2"><u class="text_2A2Ew px-1">Restore Backup Zip</u>- a plugin to current workspace</p> <h6 class="mt-1 mb-0">Help</h6><p class="mb-0"><a target="__blank" href="https://gitter.im/ethereum/remix" class="text_2A2Ew mx-1">Gitter channel</a></p><p class="mb-0"><a target="__blank" href="https://github.com/ethereum/remix-project/issues" class="text_2A2Ew mx-1">Report on Github</a></p></div>
```

Comment (LianaHus) [reply apps/remix-ide/src/app/ui/landing-page/landing-page.js:478]: I've put ballots to show that it is 2 step action not just some links

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js [hunk 2]
```diff
@@ -300,6 +301,11 @@ export class LandingPage extends ViewPlugin {
       await this.appManager.activatePlugin('pluginManager')
       this.verticalIcons.select('pluginManager')
     }
+    const startRestoreBackupZip = async () => {
+      await this.appManager.activatePlugin(['restoreBackupZip'])
+      this.verticalIcons.select('restoreBackupZip')
+      _paq.push(['trackEvent', 'pluginManager', 'userActivate', 'restoreBackupZip'])
+    }
```
Comment (bunsenstraat) [inline apps/remix-ide/src/app/ui/landing-page/landing-page.js:308]: the plugin name is lowercase 

```
    const startRestoreBackupZip = async () => {
      await this.appManager.activatePlugin(['restorebackupzip'])
      this.verticalIcons.select('restorebackupzip')
      _paq.push(['trackEvent', 'pluginManager', 'userActivate', 'restorebackupzip'])
    }
```

**Review by bunsenstraat — APPROVED** [review]

**Comment by ryestew** [comment]
There are too many different font types.
1. The Quicklinks is the headline but it doesn't read like it is.
2. 2 step solution - if I'm not sure what the problem is - I don't know what the solution is for.  Also it sounds like the name of a dance.  Also because it is bolded, it looks more important than Migrate File system.  But then what is Migrate File system? (OK I know - but it is a 1 step solution to try 1st.)
3.  Are the bullets necessary?  If it is to make a quick explanation of 2 step solution - then they should be numbered not bulleted.
4. The guide has the complete explanation and solution - but you have to read down a little to see it - (so I think it should go at the top)
5. Maybe if Quicklinks is bigger & there is a line of explaination.  Then 
What is the most important link? the Migrate File system?

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js [hunk 3]
```diff
@@ -466,9 +472,33 @@ export class LandingPage extends ViewPlugin {
...
                   <audio id="remiAudio" muted=false src="assets/audio/remiGuitar-single-power-chord-A-minor.wav"></audio>
+                  <div class="w-80 pl-5 ml-5">
+                    <h6 class="mb-1">Quicklinks</h6>
+                    <spam class="pl-0">
+                      <u class="${css.text} pr-1" onclick=${() => migrateWorkspace()}>Migrate File system</u>
+                      to workspace
+                    </spam>
+                    <p class="font-weight-bold mb-0">2-step solution:</p>
+                    <li class="pl-1">
+                      manually
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/ui/landing-page/landing-page.js:486]: `m` of `manually` can be capitalized

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js [hunk 4]
```diff
@@ -466,9 +472,33 @@ export class LandingPage extends ViewPlugin {
...
+                      <u class="${css.text} px-1" onclick=${() => downloadFiles()}>Download all Files</u>
+                      as a backup zip
+                    </li>
+                    <li class="pl-1">
+                      use
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/ui/landing-page/landing-page.js:491]: `u` of `use` can be capitalized

Comment (Aniket-Engg) [reply apps/remix-ide/src/app/ui/landing-page/landing-page.js:491]: Actually `use Restore Backup zip to current workspace` doesn't seem right. May we should remove `use`

File: apps/remix-ide/src/app/ui/landing-page/landing-page.js [hunk 5]
```diff
@@ -466,9 +472,33 @@ export class LandingPage extends ViewPlugin {
...
+                      <u class="${css.text} px-1" onclick=${() => startRestoreBackupZip()}>Restore Backup Zip</u>
+                      to current workspace
+                    </li>
+                    <p class="font-weight-bold mb-0 mt-2">Help:</p>
+                    <dir class="d-flex flex-column mt-1 pl-0">
+                      <a class="${css.text} mx-1" target="__blank" href=" https://medium.com/remix-ide/migrating-files-to-workspaces-8e34737c751c">Guide for migrating the File System to workspace</a>
```
Comment (Aniket-Engg) [inline apps/remix-ide/src/app/ui/landing-page/landing-page.js:497]: `Guide for migrating the File System to workspace` can be shortened as `Migration Guide`

**Pull request merged**